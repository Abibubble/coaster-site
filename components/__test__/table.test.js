import React from 'react'
import renderer from 'react-test-renderer'
import Table from '../table'

describe('Table', () => {
  it('should render table component', () => {
    const tree = renderer
      .create(
        <Table>
          <thead>
            <th>Test header</th>
            <th>Test header</th>
            <th>Test header</th>
          </thead>
          <tbody>
            <tr>
              <td>Test description</td>
              <td>Test description</td>
              <td>Test description</td>
            </tr>
            <tr>
              <td>Test description</td>
              <td>Test description</td>
              <td>Test description</td>
            </tr>
          </tbody>
        </Table>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
