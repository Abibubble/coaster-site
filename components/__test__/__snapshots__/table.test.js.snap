// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table should render table component 1`] = `
<table
  className="main"
>
  <thead>
    <th>
      Test header
    </th>
    <th>
      Test header
    </th>
    <th>
      Test header
    </th>
  </thead>
  <tbody>
    <tr>
      <td>
        Test description
      </td>
      <td>
        Test description
      </td>
      <td>
        Test description
      </td>
    </tr>
    <tr>
      <td>
        Test description
      </td>
      <td>
        Test description
      </td>
      <td>
        Test description
      </td>
    </tr>
  </tbody>
</table>
`;
